### PR TITLE
Tom update nao setup

### PR DESCRIPTION
**Pull request by w1211**

### CODE REVIEW

**Comment by jayenashar** [comment]
can you separate the bit at the bottom that is specific to new robots, and fix the formatting?

image

**Comment by w1211** [comment]
Unsure what you mean about that bit being specific to new robots, don't you need to do that for flashed robots as well?

**Comment by jayenashar** [comment]
After flashing a robot we don't need to create config files again, just run nao_sync -s

File: source/setup/nao_setup.rst
Comment (jayenashar) [inline source/setup/nao_setup.rst]: this needs updating

File: source/setup/nao_setup.rst
Comment (jayenashar) [inline source/setup/nao_setup.rst]: run run!

**Review by jayenashar — CHANGES_REQUESTED** [review]

File: source/setup/nao_setup.rst
Comment (jayenashar) [inline source/setup/nao_setup.rst]: probably needs the `-H nao.local $robot_name`

File: source/setup/nao_setup.rst
Comment (w1211) [reply source/setup/nao_setup.rst]: how so?

Comment (jayenashar) [reply source/setup/nao_setup.rst]: because you don't need to download an opn and flash the robot on a robot that has been factory reset

Comment (jayenashar) [reply source/setup/nao_setup.rst]: you just need nao_sync -s if it has been factory reset

File: source/setup/nao_setup.rst
Comment (w1211) [reply source/setup/nao_setup.rst]: You have my full blessing to make these changes. All this PR was trying to be was to add a one line warning - you're the one that wrote the setup stuff so this would be a far less painful experience for everyone if you made the change

Comment (jayenashar) [reply source/setup/nao_setup.rst]: can i rewrite the whole page?

Comment (w1211) [reply source/setup/nao_setup.rst]: Go for a it

File: source/setup/nao_setup.rst
Comment (jayenashar) [reply source/setup/nao_setup.rst]: it's funny.  this is the line you updated at the start, and had you updated to say this page is only for new robots, this pr probably would have been merged.

**Comment by jayenashar** [comment]
@w1211 since you made this PR you can't approve it.  just merge it when you're happy

**Comment by w1211** [comment]
🤙

**Pull request merged**
